recipes-core/images/intel-aero-image.bb

```diff
@@ -38,6 +38,9 @@ IMAGE_INSTALL += "libavahi-client libavahi-glib"
 # Build tools
 IMAGE_INSTALL += "packagegroup-core-buildessential"
 
+# Development/Debug tools
+IMAGE_INSTALL += "valgrind strace gdb cgdb"
```

Comment:
strace and gdb are already there.

valgrind, ok

cgdb would pull dependencies we currently don't have.... not sure it's worth. You can always start gdb as server and use graphical tools in your host (e.g. https://gcc.gnu.org/onlinedocs/gcc-4.8.3/gnat_ugn_unw/Remote-Debugging-using-gdbserver.html).